web/chat/threads-tab.react.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import * as React from 'react';
+
+type Props = {
+  title: string,
+  ...
```

Comment:
Can we make it exact? `{| title: string |}`

Reply:
I don't think we need this component at all, but if we were keeping it we'd want to list all of its props here. I think you're just missing `children` (which is maybe why you got errors when you tried to make it exact?)

Reply:
yes, it was because of that, but since I wasn't using `children`, I wasn't listing them